Lib/test/test_memoryview.py

```diff
@@ -545,6 +545,104 @@ def test_pickle(self):
             with self.assertRaises(TypeError):
                 pickle.dumps(m, proto)
 
+    def test_use_released_memory(self):
```

Comment:
Please mention a comment mentioning ``gh-92888: `` which explain the purpose of the test.